.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,29 @@
+name: Feature request
+description: Suggest an idea for this project
+title: 'auro-icon: [feature summary]'
+labels:
+  - 'Type: Feature'
+  - not-reviewed
+  - auro-icon
+body:
+  - type: textarea
+    id: request
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is.
+      placeholder: Ex. I'm always frustrated when [...]
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+  - type: textarea
+    id: alternative
+    attributes:
+      label: Describe alternatives you've considered
+      description: A clear and concise description of any alternative solutions or features you've considered.
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional context
+      description: Add any other context or screenshots about the feature request here.
```

Comment:
insert_final_newline

Reply:
Hi @blackfalcon ,

I will update this file as your feedback thank you.